src/main/java/ConnectionSQL.java

```diff
@@ -1,11 +1,14 @@
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class ConnectionSQL {
```

Comment:
Delete all logic from main() class it must be clear. For your database connection create another class named DBContext and create there method that will return your connection.